src/ApplicationRunner.php

```diff
@@ -52,10 +52,11 @@ public function run(): void
             $config->get('web'),
             $config->get('providers-web'),
             [],
-            null,
             $this->debug
         );
 
+        \Yiisoft\Widget\WidgetFactory::initialize($container);
```

Comment:
`yiisoft/widget` is indirect dependency (`yiisoft/yii-bulma  1.0.x-dev  requires  yiisoft/widget`). Probably need to add it to `composer.json`